lib/Habitat.lua

```diff
@@ -128,4 +128,14 @@ function Habitat:require(instance)
 	return result
 end
 
+function Habitat:loadstring(code)
+	local ServerScriptService = self.game:GetService("ServerScriptService")
+
+	if not ServerScriptService.LoadStringEnabled then
+		error("loadstring is unavailable")
+	end
+
+	return loadstring(code)
+end
```

Comment:
I'm hesitant to implement this API right now because there's still not really context between server and client scripts right now.

Reply:
No problem.